spyder/plugins/projects/api.py

```diff
@@ -94,38 +94,67 @@ def get_recent_files(self):
 
         return list(OrderedDict.fromkeys(recent_files))
 
-    def get_plugin(self, plugin_name):
-        """
-        Return a plugin by unique `plugin_name`.
-        """
-        return self.projects_plugin.get_plugin(plugin_name)
-
     # --- API
     # ------------------------------------------------------------------------
     @staticmethod
-    def get_name(self):
+    def get_name():
         """
         Provide a human readable version of NAME.
         """
         raise NotImplementedError("Must implement a `get_name` method!")
 
+    @staticmethod
+    def validate_name(path, name):
+        """
+        Validate the name of the project.

```

Comment:
Please change this to

    Validate the project's name.

Reply:
Fixed